src/Containers/msgtitle.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Msgtitle() {
+  return (
+    <div className="Msgtitle">
+      <span className="title fw-700">All your steps, One steps place</span>
+      <span className="desc">Keep track over all your books just by adding those with just one key</span>
+    </div>
+  );
+}
+
+export default Msgtitle;
```

Comment:
This seems like a presentational component, should be on components directory.

Reply:
Thanks for the notes I think I got to save them xD @MauricioRobayo. they will be usefull 🚀.